dallinger/default_configs/global_config_defaults.txt

```diff
@@ -52,4 +52,4 @@ prolific_is_custom_screening = False
 prolific_recruitment_config = {}
```

Comment:
Please could you update the PR description to explain in a little more detail what the problem was and how your fix addresses it?

Reply:
Will do it when all other of your comments are resolved.

Reply:
Looks good! just this one left

Reply:
Done. Hope it is clearer now.